package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "next-axiom",
   "description": "Send WebVitals from your Next.js project to Axiom.",
-  "version": "0.15.1",
+  "version": "0.16.1",
```

Comment:
I don't think this or the README.md changes were intended to be in this PR, were they?

Reply:
the package.json was, looks like I made mistake with the README. I will remove both of them from this PR.